### ISSUE TITLE
Let users create MQFs

### ISSUE THREAD

**Issue by cbrech** [airmencoders/MQF#7]
How should we let users create and distribute their own MQFs?

This is related to issue #6 however that issue is about getting the data into the app. This issue is about generating the data. 

Ideally we would have a system people could log into, build an MQF and save/share it with others. Currently we don’t have a hosting environment so looking for ideas for the interim.

**Comment by airmencoders** [comment]
Thanks to a great idea by @chris-m92 I think we will be building a static website that lets people create and edit their MQFs. They would have to save a file at the end of each session to prevent us from storing FOUO on our servers. Thoughts?

**Comment by chris-m92** [comment]
That could work, exporting a JSON file for them to upload or, I'm working
on a contact to get a hosting solution on DoD servers that can securely
host the information at the appropriate levels.

Problem then turns to great, we can host the data but then how do restrict
access and authenticate users on the app.

**Comment by airmencoders** [comment]
Yeah, one problem at time I guess. If we build the editor now it shouldn't be that hard to add a save function down the road.

**Comment by chris-m92** [comment]
Once I get off shift today I'll start working on my proof of concept,
should be easy enough to do.

**Comment by cralor** [comment]
In the mean time, what's the best way to contribute MQF content?

**Comment by airmencoders** [comment]
@cralor use this file: https://github.com/airmencoders/MQF/blob/master/template.xls and send it to me, then I will import it
